Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -4290,8 +4290,14 @@ - (void)_requestTextExtraction:(CGRect)rectInWebView completionHandler:(void(^)(
 #endif
     }();
 
-    _page->requestTextExtraction(WTFMove(rectInRootView), [completionHandler = makeBlockPtr(completionHandler)](auto&& item) {
-        completionHandler(WebKit::createItem(item).get());
+    _page->requestTextExtraction(WTFMove(rectInRootView), [completionHandler = makeBlockPtr(completionHandler), weakSelf = WeakObjCPtr<WKWebView>(self)](auto&& item) {
+        completionHandler(WebKit::createItem(item, [strongSelf = weakSelf.get()](auto& rectInRootView) {
+#if PLATFORM(IOS_FAMILY)
+            if (RetainPtr contentView = strongSelf ? strongSelf->_contentView : nil)
+                return WebCore::FloatRect { [strongSelf convertRect:rectInRootView fromView:contentView.get()] };
```

Comment:
I think you can skip `WebCore::FloatRect { }`, right? Since this isn't marked `explicit`.

`WEBCORE_EXPORT FloatRect(const NSRect&);`

Reply:
(Note: I needed to explicitly add a `-> WebCore::FloatRect` for the lambda, otherwise the return value can't be implicitly deduced)